features/map_test.go

```diff
@@ -64,9 +65,25 @@ func TestHaveMapType(t *testing.T) {
 				t.Fatalf("Map type %s isn't supported even though kernel is at least %s: %v", mt.String(), minVersion, err)
 			}
 		})
-
 	}
+}
+
+var mapFlagMinVersion = map[mapFlagCacheEntry]string{
+	{ebpf.Array, unix.BPF_F_MMAPABLE}: "5.5",
```

Comment:
We should cover more combinations with tests and test failing outcomes as well.

Reply:
Added more tests, covering the internal feature tests functions